uaclient/config.py

```diff
@@ -50,7 +50,10 @@ def __init__(self, cfg=None):
     @property
     def accounts(self):
         """Return the list of accounts that apply to this authorized user."""
-        return self.read_cache('accounts')['accounts']
+        accounts = self.read_cache('accounts')
+        if not accounts:
+            return []
+        return accounts['accounts']
```

Comment:
I wonder if we should be catching `KeyError` here and raising an exception with a better error, in case the accounts cache file ends up incorrect (it could be empty if it was written when the disk was full, for example).

(We could also consider checking that `accounts['accounts']` is a list, to avoid potentially very confusing errors elsewhere in the codebase.)